src/resources/images/licenses/open-access-logo.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg" width="640" height="1000" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:cc="http://creativecommons.org/ns#" xmlns:dc="http://purl.org/dc/elements/1.1/">
+  <metadata><rdf:RDF><cc:Work rdf:about="">
+    <dc:format>image/svg+xml</dc:format>
+    <dc:type rdf:resource="http://purl.org/dc/dcmitype/StillImage"/>
+    <dc:creator>art designer at PLoS, modified by Wikipedia users Nina, Beao, JakobVoss, and AnonMoos</dc:creator>
+    <dc:description>Open Access logo, converted into svg, designed by PLoS. This version with transparent background.</dc:description>
+    <dc:source>http://commons.wikimedia.org/wiki/File:Open_Access_logo_PLoS_white.svg</dc:source>
+    <dc:license rdf:resource="http://creativecommons.org/licenses/by-sa/3.0/"/>
+    <cc:license rdf:resource="http://creativecommons.org/licenses/by-sa/3.0/"/>
```

Comment:
`You must give appropriate credit, provide a link to the license, and indicate if changes were made.`
Må vel være greit når all data ligger i fila?

Reply:
det vil jeg mene